### PR TITLE
added github link and removed alerts

### PR DESCRIPTION
**Pull request by viveksinghggits**
This change fixes issues #15 and #16

### CODE REVIEW

**Comment by viveksinghggits** [comment]
Hi @PrasadG193 ,
Dont merge this yet, found an issue.

**Comment by viveksinghggits** [comment]
@PrasadG193  It can be merged now.

File: frontend/index.html [hunk 1]
```diff
@@ -55,6 +62,12 @@
       </div>
       <hr>
     </div> <!-- /container -->
+    <div id="footer">
+        <span id="gith">
+            <img src="image/GitHub-Mark-64px.png" id="gitlogo">
```
Comment (PrasadG193) [inline frontend/index.html]: Please use font-awesome icons

**Comment by PrasadG193** [comment]
Please move the github button to top-right corner

**Comment by PrasadG193** [comment]
Thanks for the PR @viveksinghggits

**Pull request merged**